### ISSUE TITLE
Provide POSIX script for account_command

### ISSUE THREAD

**Issue by mervn** [neomutt/neomutt#4458]
## What would you like to be able to do?

Use a POSIX compliant script to retrieve information for "account_command". The script must only contain commands present in the POSIX.1-2017 standard. This script should support all valid options (i.e --username) and all valid outputs (i.e login: ).

## What needs to change in NeoMutt?

- Add the file /data/account-command/posix/account.sh - the POSIX script.  
- In the README, state that the only dependency is awk(1) - which should be installed on POSIX machines **but might not be**.

Being portable is paramount, the script should only contain commands present in POSIX.1-2017. [I have written an implementation](https://gist.github.com/mervn/a54235b70df01ff5c65352dcdc4d9dec) which has the same license as NeoMutt.

## Have you tried any alternative methods?

No. "account_command" does not have an equivalent alternative.

## Additional Info

[There is an implementation for himitsu users](https://git.sr.ht/~whynothugo/himitsu-mutt).

The configuration section of my implementation was inspired (and is compatible) with this.

**Comment by gahr** [comment]
I am sorry, I totally don't get it.. what would the script do with the information? The one you have written uses some [accountrc spec](https://gist.github.com/mervn/a54235b70df01ff5c65352dcdc4d9dec#file-account-sh-L78-L85), which honestly I've never heard about. The one from himitsu, well, uses himitsu. There's not much in common... 
The [parsing part](https://git.sr.ht/~whynothugo/himitsu-mutt/tree/main/item/himitsu-mutt#L21-43) is all there *could* be in common, but then each backend would use those info in a different way. So yeah, I'm confused.. what are you looking for?

**Comment by gahr** [comment]
> TL;DR. account_command is not usable out-of-the-box, this implementation allows for it to be used with minimal setup. I would like to contribute this script for those who are looking for something similar.

I understand your solution fits your use case. I am not sure we want to advertise it as *the* way to interact with account_command, or present it as the official protocol between account_command and the user.

(In my opinion, we don't need another protocol: you'd generally parse the arguments of account_command in 10 lines or less and then hand it off to your password management logic, which can be more flexible than what your config format would be best fit for.)

Having said that, I don't have a big appreciation of how users are using account_command out there, and if you think your solution is generally used/useful, I think we wouldn't be opposed to a script and a readme in a sub directory in contrib/. Feel free to open a PR and we can discuss the details there.

**Comment by gahr** [comment]
>> Having said that, I don't have a big appreciation of how users are using account_command out there

> We'll likely discuss this in the PR, but I am interested in your perspective. What does this mean?

Oh, this just means that I don't know whether, how, and how much the feature is used.
